runtime/oti/j9nonbuilder.h

```diff
@@ -2989,13 +2996,19 @@ typedef struct J9IndexableObjectDiscontiguous {
 #if defined(OMR_GC_COMPRESSED_POINTERS) || !defined(J9VM_ENV_DATA64)
 	U_32 padding;
 #endif /* OMR_GC_COMPRESSED_POINTERS || !J9VM_ENV_DATA64 */
+#if defined(J9VM_ENV_DATA64)
```

Comment:
Same comment as above - don't change the unused structures.

Reply:
Those types *are* used in DDR code, e.g. [J9IndexableObjectHelper](https://github.com/eclipse/openj9/blob/master/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java#L113).